src/main/resources/META-INF/plugin.xml

```diff
@@ -12,6 +12,7 @@
         <changesViewContent tabName="Git Machete"
                             className="com.virtuslab.gitmachete.frontend.ui.impl.root.GitMacheteContentProvider"
                             predicateClassName="com.virtuslab.gitmachete.frontend.ui.impl.root.GitMacheteVisibilityPredicate"/>
+        <postStartupActivity implementation="com.virtuslab.gitmachete.frontend.ui.impl.root.GitMacheteStartupActivity"/>
```

Comment:
Nice! So this will run after:
1. the plugin classes are loaded
or later when
2. the tab is opened?

Reply:
this one will run on classes loading and add a listener for tab open